ncat/ncat_core.c

```diff
@@ -60,7 +60,7 @@
  * OpenSSL library which is distributed under a license identical to that  *
  * listed in the included docs/licenses/OpenSSL.txt file, and distribute   *
  * linked combinations including the two.                                  *
- *                                                                         * 
+ *                                                                         *
```

Comment:
These whitespace changes will be fixed if you rebase the patch; this might require some extra work, because I couldn't get the patch to apply cleanly, but it's something you'd have to do anyway before merging in svn.